### DISCUSSION TITLE
error[E0432]: unresolved import `pyo3::types::PyDictItems`

### DISCUSSION THREAD

**Discussion by gabriel-frattini** [PyO3/pyo3#3085] [Questions]
In the PyO3/maturin-action@v1 pipeline I get an error on line 812 [here](https://github.com/gabriel-frattini/aluminum/actions/runs/4636183575/jobs/8203845028#step:3:818). 

it says `unresolved import pyo3::types::PyDictItems` and `no PyDictItems in types`. Why can't it find PyDictItems?

**Comment by messense** [comment]
[`PyDictItems`](https://docs.rs/pyo3/latest/pyo3/types/struct.PyDictItems.html) isn't available on PyPy.

**Reply by gabriel-frattini** [reply]
Okay is there an alternative way of writing this that does not cast "items" as PyDictItems?


```

let items = cls
            .getattr("__annotations__")?
            .getattr("items")?
            .call0()?
            .downcast::<PyDictItems>()?;
```
